### PR TITLE
Minor fixes to `D.Uniform`

### PR DESCRIPTION
**Pull request by toslunar**

### CODE REVIEW

File: chainer/distributions/uniform.py [hunk 1]
```diff
@@ -75,33 +76,32 @@ def batch_shape(self):
...
         xp = cuda.get_array_module(x)
 
-        logp = -exponential.log(bh - bl)
+        logp = broadcast.broadcast_to(
+            -exponential.log(self.scale), x.shape)
         return where.where(
-            xp.asarray((x.data >= bl.data) & (x.data < bh.data)),
-            logp, xp.asarray(-xp.ones_like(x.data)*numpy.inf, dtype=x.dtype))
+            utils.force_array(
+                (x.data >= self.low.data) & (x.data < self.high.data),
```
Comment (okuta) [inline chainer/distributions/uniform.py]: Please remove comma.

File: chainer/distributions/uniform.py [hunk 2]
```diff
@@ -75,33 +76,32 @@ def batch_shape(self):
         return self.low.shape
 
     def cdf(self, x):
-        return clip.clip((x - self.low)/(self.high - self.low), 0., 1.)
+        return clip.clip((x - self.loc)/self.scale, 0., 1.)
```
Comment (okuta) [inline chainer/distributions/uniform.py]: flake8

**Comment by toslunar** [comment]
Fixed.

**Comment by okuta** [comment]
jenkins, test this please.

**Comment by okuta** [comment]
LGTM!

**Pull request merged**